src/visual.ts

```diff
@@ -447,29 +462,44 @@ module powerbi.extensibility.visual {
             );
 
             this.xAxisProperties.formatter = this.data.dateColumnFormatter;
+            let yMin = this.data.yMinValue,
+                yMax = this.data.yMaxValue;
+            if (yMax === yMin) {
+                if ((Math.floor(yMin) === yMin) && yMin !== 0) {
+                    yMin = yMin - 1;
+                    yMax = yMax + 1;
+                } else {
+                    yMin = yMin - LineDotChart.dateMaxCutter;
+                    yMax = yMax + LineDotChart.dateMaxCutter;
+                }
+            } else {
+                yMax = yMax + (yMax - yMin) * LineDotChart.makeSomeSpaceForCounter;
+            }
 
             this.yAxisProperties = AxisHelper.createAxis({
                 pixelSpan: effectiveHeight,
-                dataDomain: [this.data.yMinValue, this.data.sumOfValues],
+                dataDomain: [yMin, yMax],
                 metaDataColumn: this.data.valuesMetadataColumn,
                 formatString: null,
                 outerPadding: LineDotChart.outerPadding,
                 isCategoryAxis: false,
                 isScalar: true,
                 isVertical: true,
-                useTickIntervalForDisplayUnits: true
+                useTickIntervalForDisplayUnits: true,
+                getValueFn: valueFormattingFn
             });
 
             this.yAxis2Properties = AxisHelper.createAxis({
                 pixelSpan: effectiveHeight,
-                dataDomain: [this.data.yMinValue, this.data.sumOfValues],
+                dataDomain: [yMin, yMax],
```

Comment:
IF we are going to publish it we should increase versions and describe changes in CHANGELOG.md